features/machine/machine.feature

```diff
@@ -359,10 +359,9 @@ Feature: Machine features testing
       | failure-domain.beta.kubernetes.io/region in [<%= cb.default_region %>] |
 
   # @author miyadav@redhat.com
-  # @case_id OCP-33380
   @admin
   @destructive
-  Scenario: Implement defaulting machineset values for vsphere
+  Scenario Outline: Implement defaulting machineset values for vsphere
```

Comment:
It's a scenario outline now, no longer specific to vsphere. Perhaps remove `for vsphere` here ?